README.md

```diff
@@ -1,14 +1,117 @@
+  </a>
+  <h2 align="center">ForgeRock SDK UI for JavaScript</h2>
+  <p align="center">
+    <a href="https://github.com/ForgeRock/forgerock-javascript-sdk-ui/blob/master/CHANGELOG.md">Change Log</a>
+    ·
+    <a href="#support">Support</a>
+    ·
+    <a href="#documentation" target="_blank">Docs</a>
+  </p>
+<hr/></p>
+
+The ForgeRock JavaScript SDK with UI contains the [Core SDK](https://github.com/ForgeRock/forgerock-javascript-sdk) and adds UI rendering capability. This project greatly reduces the effort to quickly prototype and demonstrate the use of the Core SDK in a more real-word scenario.
+
+<!------------------------------------------------------------------------------------------------------------------------------------>
+<!-- REQUIREMENTS - Supported AM versions, API versions, any other requirements. -->
+
+## Requirements
+
+* ForgeRock Identity Platform
+    * Access Management (AM) 6.5.2+
+
+* Browsers:
+    * Edge 44 (Legacy): requires one polyfill for TextEncoder, [`fast-text-encoding` is recommended](https://www.npmjs.com/package/fast-text-encoding)
+    * Edge 83 (Chromium)
+    * Chrome 83
+    * Firefox 77
+
+> **Tip**: Older browsers (like IE11) may require multiple [polyfills, which can be found in our documentation](https://sdks.forgerock.com/javascript/polyfills/).
+
+<!------------------------------------------------------------------------------------------------------------------------------------>
+<!-- INSTALLATION -->
 
 ## Installation
 
-```bash
+```
 npm install @forgerock/javascript-sdk-ui
 ```
 
-## Concepts
+<!------------------------------------------------------------------------------------------------------------------------------------>
+<!-- QUICK START - Get one of the included samples up and running in as few steps as possible. -->
+
+## Getting Started
+
+To try out the ForgeRock JavaScript SDK UI, perform these steps:
+
+1. Setup CORS support in an Access Management (AM) instance.
+
+   See [Enabling CORS Support](https://sdks.forgerock.com/js/01_prepare-am/#enabling-cors-support) in the Documentation.
+
+2. Create an authentication tree in AM.
+
+   See [Creating a User Authentication Tree](https://sdks.forgerock.com/js/01_prepare-am/#creating-a-user-authentication-tree) in the Documentation.
+
+3. Clone this repo:
+
+    ```
+    git clone https://github.com/ForgeRock/forgerock-javascript-sdk-ui.git
+    ```
+
+4. In the root folder of the repo, use NPM to install dependencies:
+
+    ```
+    npm install
+    ```
+
+5. Build the ForgeRock JavaScript SDK:
+
+    ```
+    npm run build
+    ```
+
+6. Open `samples/basic/index.html` and edit the configuration values to match your AM instance.
+
+7. You will need to access the sample app with HTTPS (secure protocol). To do that, you'll need to create your own trusted certs. You are free to use any method to do this, but if you need assistance in generating certs, the following can be helpful:
```

Comment:
The term "trusted certs" here worries me a bit.  Specifically, I'm concerned that referring to them as "trusted" could lead the reader to believe that using a self-signed cert is an acceptable practice in general.

To be clear, using a self-signed cert here -- as part of sample code that people can use to learn the platform -- is totally fine.  But we should advise people against doing this in production.  (Some of them may do it anyway, but let's make sure we're not to blame :) )

Perhaps the way to finesse it would be as follows:
- The product requires a trusted certificate
- For learning/lab/testing purposes, you can create a self-signed cert and tell the product to trust it.
- This is not a good practice for production systems.  Buy a real cert!  Issue one under your corporate CA!  Or something!

Reply:
Thanks for looking over this, @ScottCulp. Very good idea! I will make this change.